README.md

```diff
@@ -1,3 +1,17 @@
 <img src="https://arweave.net/dOpRkKrNNQ4HHebxZlPCo0BWfrjwJ-CEBQs2EPgrwbg" />
 
-Soon.
\ No newline at end of file
+## Identify bottlenecks with eflame
```

Comment:
We should leave the readme as is for now. This seems like a good segment to put into the hacking guide in docs.

Reply:
Ok, I have moved the copy to the `hacking-on-hyperbeam` section